common/cosmwasm-smart-contracts/mixnet-contract/src/gateway.rs

```diff
@@ -6,10 +6,11 @@ use cosmwasm_schema::cw_serde;
 use cosmwasm_std::{Addr, Coin};
 use std::cmp::Ordering;
 use std::fmt::Display;
+use utoipa::ToSchema;
 
 /// Information provided by the node operator during bonding that are used to allow other entities to use the services of this node.
 #[cw_serde]
-#[derive(PartialOrd)]
+#[derive(PartialOrd, ToSchema)]
```

Comment:
can you feature-lock utoipa in here? the problem with that type (and everything in `common/cosmwasm-smart-contracts`) is that it's pulled by our smart contracts which creates 2 problems:
- bloats the contract sizes
- (not sure if that's the case in this instance) possibly makes contracts incompatible by introducing float operations. for example some serde's features don't work correctly.

you can see an example of feature-locking utoipa here: https://github.com/nymtech/nym/blob/10ff165c187dd04180442fe126dceac2db39a644/common/exit-policy/src/policy/address_policy.rs#L238

Reply:
and I guess this comment will apply to everything in this crate

Reply:
makes sense, I'll feature-guard it